bugalink-frontend/pages/ride/[id]/details.tsx

```diff
@@ -0,0 +1,184 @@
+          <div className="py-2">
+            <p className="text-sm text-gray">Fecha y hora</p>
+            <p className="text-md text-justify font-medium">
+              📅 Todos los {dayRoutineText} a las{' '}
+              {/*TODO El toLocaleString creo que está sumando dos horas REVISAR*/}
+              {startTime.toLocaleString('es-ES', {
+                hour: '2-digit',
+                minute: '2-digit',
+              })}
+            </p>
+          </div>
+          <div className="py-2">
+            <p className="text-sm text-gray">Sobre el conductor</p>
+            <p className="text-md text-justify font-medium">
+              {preferences?.prefers_talk
+                ? '🗣 Prefiero hablar durante el viaje'
+                : '🤫 Prefiero no hablar durante el viaje'}
+            </p>
+            <p className="text-md text-justify font-medium">
+              {preferences?.prefers_music
+                ? '🎶 Prefiero ir escuchando música'
+                : '🔇 Prefiero ir sin música'}
+            </p>
+            <p className="text-md text-justify font-medium">
+              {preferences?.allows_pets
+                ? '🐾 Acepto mascotas'
+                : '😿 No acepto mascotas'}
+            </p>
+            <p className="text-md text-justify font-medium">
+              {preferences?.allows_smoke
+                ? '🚬 Se puede fumar en el coche'
+                : '🚭 No fumar en el coche'}
+            </p>
+          </div>
+          {trip.driver_routine.note && (
+            <div className="py-2">
+              <p className="text-sm font-normal text-gray">
+                Nota del conductor
+              </p>
+              <p className="text-md text-justify font-medium leading-5">
+                ✏️ {trip.driver_routine.note}
+              </p>
+            </div>
+          )}
+        </div>
+        {/* Trip request */}
+        <div className="w-full rounded-t-xl bg-white py-6 px-5 shadow-t-md">
+          <div className="flex flex-row items-center justify-between">
+            <div className="flex flex-col">
+              <p className="text-sm font-normal">Tipo de viaje</p>
+              <p className="text-xl font-bold">
+                {trip?.driver_routine.is_recurrent ? 'Recurrente' : 'Único'}
+              </p>
+            </div>
+            <div className="flex flex-col">
+              <p className="text-sm font-normal">Precio por asiento</p>
+              <p className="text-xl font-bold">{trip.driver_routine.price}€</p>
+            </div>
+            <div className="flex flex-col">
+              <p className="text-sm font-normal">Plazas libres</p>
+              <p className="text-xl font-bold">
+                {trip.driver_routine.available_seats}
+              </p>
+            </div>
+          </div>
+          {requested === 'false' && (
+            <div className="flex justify-center">
+              <Link
+                href="/ride/V1StGXR8_Z5jdHi6B-myT/detailsTwo"
```

Comment:
This part of code (appearing when query param of `details?requested=false`) has not been adapted to the new structure without `detailsTwo`. It should be named something like "PAGAR" and with an href linking to the ride payment. Take as an example the behavior in the deleted `detailsTwo.tsx`

Reply:
![image](https://user-images.githubusercontent.com/80245816/230373818-2c4ee54d-68d5-4bbe-a6d5-b8168aa71f4a.png)